cmd/cluster-agent/api/v1/install.go

```diff
@@ -0,0 +1,152 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/dd-go/log"
+	"github.com/gorilla/mux"
+)
+
+func Install(r *mux.Router) {
+	r.HandleFunc("metadata/nodes/{nodeName}/namespaces/{ns}/pods/{podName}", getPodMetadata).Methods("GET")
+	r.HandleFunc("metadata/nodes/{nodeName}", getNodeMetadata).Methods("GET")
+	r.HandleFunc("metadata", getAllMetadata).Methods("GET")
+	r.HandleFunc("{check}/events", getCheckLatestEvents).Methods("GET")
+}
+
+// TODO: complete it
```

Comment:
I think this can be dropped at the moment ? 
I wanted to remove the TODOs at some point, I guess it's a good time now